media/filters/chunk_demuxer.cc

```diff
@@ -386,6 +464,76 @@ void ChunkDemuxerStream::ChangeState_Locked(State state) {
 
 ChunkDemuxerStream::~ChunkDemuxerStream() = default;
 
+#if defined(STARBOARD)
+void ChunkDemuxerStream::CompletePendingReadIfPossible_Locked() {
```

Comment:
This is introduced in CL 231440 to support batch writing.  Now batch writing is supported by default, so likely this can be removed (as well as `pending_config_change_`).  Feel free to keep this as is and create a bug to remove immediately after merging.

Reply:
I removed this customization.